frameworks/hdfs/src/main/dist/modify-keytab.sh

```diff
@@ -0,0 +1,3 @@
+export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+$JAVA_HOME/bin/java -jar krbtest.jar secrets-hdfs.keytab 
+echo "Keytab file modified"
```

Comment:
Can you change this message to something that describes the fix and reference the HDFS issue? eg
```
echo "Keytab file fixed due to bug in HDFS version 3.2.0. See HDFS#123 http://apache.hdfs<url_link>
```
That way customers know what's happening and can see why we're modifying their keytabs rather than be cryptic about it. Once HDFS releases a new version including the fix we can later delete this script.

Can you also rename the file to something like `fix-keytab-bug.sh`

Reply:
Please add a shebang to the script and `set -x`.

Reply:
Also, could you name the script something more specific with regards to its intent? "modify" is very vague imo.

Reply:
Done